modules/p2/ting_openlist/ting_openlist.module

```diff
@@ -304,7 +304,7 @@ function ting_openlist_get_soap($wsdl = NULL, $trace = FALSE) {
  * @return mixed
  *   Response from Openlist.
  */
-function ting_openlist_request($function, array $arguments = array(), $detailed_log = FALSE) {
+function ting_openlist_request($function, array $arguments = array(), $detailed_log = FALSE, $error_log = TRUE) {
```

Comment:
This adds a new parameter `$error_log`. If this approach stays then this should be documented.